src/services/TagsService.js

```diff
@@ -150,12 +150,12 @@ export default class TagsService {
    *
    * @return {boolean}
    */
-  containsKeyword(str) {
+  containsAnyKeyword(str) {
     if (_.isArray(str)) {
```

Comment:
You can change it on [Array.isArray](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)